raiden/ui/console.py

```diff
@@ -37,13 +36,10 @@
 OKGREEN = "\033[92m"
 ENDC = "\033[0m"
 
-# ipython needs to accept "--gui gevent" option
-IPython.core.shellapp.InteractiveShellApp.gui.values += ("gevent",)
```

Comment:
This works without all that gevent stuff now?

Reply:
Yes, from 5.0 onwards IPython stopped using gnureadline (which was the point of all this) and instead uses prompt-toolkit which is pure python. Therefore it seems that gevent monkeypatching makes it work.